tari-project/universe
# feat(ui): temp paper wallet QR code

Description
---
- quick QR code implementation so testing on aurora can start


How Has This Been Tested?
---
- ~~locally with testflight aurora, though nothing happened after the app opened (assuming because the local network  is esmeralda and aurora is on nextnet)~~ see comment

<img width="1170" alt="Screenshot 2024-10-15 at 15 45 32" src="https://github.com/user-attachments/assets/ad937d9d-e4e3-4b76-98bc-1d4a01255218">


What process can a PR reviewer use to test or verify this change?
---
- set the `paper_wallet_enabled` flag 
- load and scan the QR code in wallet settings
